internal/cli/server/config.go

```diff
@@ -146,6 +146,12 @@ type Config struct {
 
 	// HistoryConfig has historical data retention related settings
 	History *HistoryConfig `hcl:"history,block" toml:"history,block"`
+
+	// VMTrace Name of tracer which should record internal VM operations (costly)
```

Comment:
These two fields are added in the `Config` structure, and I see that you have added the flags in the `[log]` section of the config file. This will not work.

I would recommend creating another structure/section for this, or move them to the one which is most relevant.